File: worker-java/FixAutoIncrement.java
import java.io.*;
import java.nio.file.*;
import java.util.stream.*;

public class FixAutoIncrement {
    public static void main(String[] args) throws Exception {
        Path startDir = Paths.get("c:/Users/ruanp/NETBENAS - COWORK/migrador-web/worker-java/src/br/com/lcsistemas");
        Files.walk(startDir)
             .filter(Files::isRegularFile)
             .filter(p -> p.toString().endsWith(".java"))
             .forEach(p -> {
                 try {
                     String content = new String(Files.readAllBytes(p), "UTF-8");
                     if (content.contains("AUTO_INCREMENT = 1\"")) {
                         String newContent = content.replaceAll("(?m)^(\\s*)(exec.*AUTO_INCREMENT = 1.*)$", "$1// $2");
                         if (!content.equals(newContent)) {
                             Files.write(p, newContent.getBytes("UTF-8"));
                             System.out.println("Fixed: " + p.getFileName());
                         }
                     }
                 } catch (Exception e) {
                     e.printStackTrace();
                 }
             });
    }
}

File: worker-java/fix_auto_increment.py
import os
import re

src_dir = r"c:\Users\ruanp\NETBENAS - COWORK\migrador-web\worker-java\src\br\com\lcsistemas"

for root, dirs, files in os.walk(src_dir):
    for file in files:
        if file.endswith(".java"):
            filepath = os.path.join(root, file)
            with open(filepath, 'r', encoding='utf-8') as f:
                content = f.read()
            
            # replace lines like: execIgnore(my, "ALTER TABLE lc_sistemas.ncm AUTO_INCREMENT = 1", "reset AI ncm");
            # by commenting them out. Some are split across lines, but wait, the grep output showed them.
            # Let's just match any line that contains AUTO_INCREMENT = 1 and if it doesn't start with //, prepend //
            
            lines = content.split('\n')
            modified = False
            for i, line in enumerate(lines):
                if 'AUTO_INCREMENT = 1' in line and not line.strip().startswith('//'):
                    lines[i] = '// ' + line
                    modified = True
            
            if modified:
                with open(filepath, 'w', encoding='utf-8') as f:
                    f.write('\n'.join(lines))
                print("Modified", filepath)
